torch/_inductor/decomposition.py

```diff
@@ -304,6 +304,12 @@ def bernoulli(self, *, generator=None):
     return torch.rand_like(self, dtype=torch.float32) < self
 
 
+@register_decomposition([aten.bernoulli.p])
+def bernoulli(self, p=0.5, *, generator=None):
+    assert generator is None
+    return torch.rand_like(self, dtype=torch.float32) < p
```

Comment:
I did not put this in the real decomp bin because I followed the pattern in bernoulli, but the decomp looks a bit questionable...